docs/user/customization.md

```diff
@@ -64,4 +64,106 @@ An example `worker.ign` is shown below. It has been modified to increase the HTT
+
+1. `openshift-install create manifests --dir $INSTALL_DIR`
+
+2. Copy files with `machineconfig` objects to `$INSTALL_DIR/openshift/`
+
+3. `openshift-install create cluster --dir $INSTALL_DIR`
+
+These custom `machineconfig` objects are black boxes to the installer and the installer only plays the role of `oc create -f <custom-machine-config-object>` early enough into cluster bootstrap to make sure the configuration is used by the [MachineConfigOperator][machine-config-operator].
+
+For example, all control plane nodes register with taint `node-role.kubernetes.io/master=:NoSchedule` making them unschedulable by most normal workloads. An imaginary installation that requires the control plane to boot without that taint can push a custom `machineconfig` with `kubelet.service` that doesn't include the taint.
+
+On Libvirt:
+
+1. Run `manifests` target to create all the manifests.
+
+```console
+ls -l no-taint-cluster/**
+```
+
+2. Create a `machineconfig` that includes `kubelet.service` that has no taints.
+
+```console
+$ cat > no-taint-cluster/openshift/99-master-kubelet-no-taint.yaml <<EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 02-master-kubelet
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    systemd:
+      units:
+      - contents: |
+          [Unit]
+          Description=Kubernetes Kubelet
+          Wants=rpc-statd.service
+
+          [Service]
+          Type=notify
+          ExecStartPre=/bin/mkdir --parents /etc/kubernetes/manifests
+          ExecStartPre=/bin/rm -f /var/lib/kubelet/cpu_manager_state
+          EnvironmentFile=-/etc/kubernetes/kubelet-workaround
+          EnvironmentFile=-/etc/kubernetes/kubelet-env
+
+          ExecStart=/usr/bin/hyperkube \
+              kubelet \
+                --config=/etc/kubernetes/kubelet.conf \
+                --bootstrap-kubeconfig=/etc/kubernetes/kubeconfig \
+                --rotate-certificates \
+                --kubeconfig=/var/lib/kubelet/kubeconfig \
+                --container-runtime=remote \
+                --container-runtime-endpoint=/var/run/crio/crio.sock \
+                --allow-privileged \
+                --node-labels=node-role.kubernetes.io/master \
+                --minimum-container-ttl-duration=6m0s \
+                --client-ca-file=/etc/kubernetes/ca.crt \
+                --volume-plugin-dir=/etc/kubernetes/kubelet-plugins/volume/exec \
+                \
```

Comment:
nit: probably don't need this line ;)

Reply:
want to keep the service file as close to the default.

Reply:
> want to keep the service file as close to the default.

Fair enough.  I've [asked after it upstream][1], but tracking whatever they do here makes sense.

[1]: https://github.com/openshift/machine-config-operator/pull/213/files#r261430402